rasa/core/training/interactive.py

```diff
@@ -168,19 +171,34 @@ class Abort(Exception):
                                       subpath=subpath)
     except ClientError:
         if is_new_action:
-            warning_questions = questionary.confirm(
-                "WARNING: You have created a new action: '{}', "
-                "which was not successfully executed. "
-                "If this action does not return any events, "
-                "you do not need to do anything. "
-                "If this is a custom action which returns events, "
-                "you are recommended to implement this action "
-                "in your action server and try again."
-                "".format(action_name))
-            await _ask_questions(warning_questions, sender_id, endpoint)
+            if action_name in NEW_TEMPLATES:
+                warning_questions = questionary.confirm(
+                    "WARNING: You have created a new action: '{0}', "
+                    "with matching template: {1}. "
+                    "This action will not return its message in this session, "
+                    "but the new utterance will be saved to your domain file "
+                    "when you exit and save this session. "
+                    "You do not need to do anything further. "
+                    "If you want to change the utterance or implement '{0}' as "
+                    "a custom action you are recommended to implement this "
+                    "in your domain file/ action server and try again. "
+                    "".format(action_name, [*NEW_TEMPLATES[action_name]]))
```

Comment:
is the idea here that you can add multiple templates at once? otherwise why are we formatting it as a list?
```
? Please type the action name utter_hello                                                        
? Please type the message for your new action utter_hello Hi                                     
Thanks! The bot will now run utter_hello.
```
For reference ^ it only prompts me for one

Reply:
I used the list so i can use starred expression to get the keys

Reply:
`? WARNING: You have created a new action: 'utter_hello', with matching template: ['Hi'].` displaying it in a list is weird though

Reply:
ok, I can remove the list for the display